tasks/dialin.yml

```diff
@@ -0,0 +1,7 @@
+---
+
+- include: dialin_standalone.yml
+  when: bbb_sipgw is not defined
+
+- include: dialin_sipgw.yml
+  when: bbb_sipgw is defined
```

Comment:
Since the var `bbb_sipgw` is not defined by default this will only get deployed when the users sets it.
This needs documentation.